bower.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "jquery-initialize",
-  "version": "1.3.0",
+  "version": "1.4.0",
   "description": "jQuery plugin for dynamically created elements initialization",
   "main": "jquery.initialize.js",
   "authors": [
-    "timpler"
```

Comment:
come on man

Reply:
Hi,
I am so sorry, I just wanted to change the URL. It looks like I replaced all occurrences of "timpler". I rolled back with new commits.